website/src/components/features.tsx

```diff
@@ -0,0 +1,177 @@
+import { Tab } from "@headlessui/react";
+import { MDXProvider } from "@mdx-js/react";
+import clsx from "clsx";
+import Highlight, { defaultProps } from "prism-react-renderer";
+import React, { Fragment } from "react";
+
+import { Code } from "./code";
+import FunctionlessTableFunction from "./snippets/functionless-table-function.mdx";
+
+export const Features = () => {
+  return (
+    <section className="py-36 bg-functionless-bg-alternate dark:bg-functionless-dark-bg-alternate home-features">
+      <div className="max-w-screen-md container flex flex-col items-center">
+        <span className="over">FEATURES</span>
+        <h3 className="text-center mt-2">
+          Build Reliable and Scalable systems with Smart Resources.
+        </h3>
+      </div>
+      <div className="grid grid-cols-1 md:grid-cols-2 container max-w-screen-xl py-36 gap-11">
+        <div>
+          <h4>Code</h4>
+          <div className="mt-10">
+            <h5 className="m-0">Infrastructure from code</h5>
+            <p className="body1 text-functionless-medium dark:text-functionless-dark-medium mt-2">
+              Build functionless architectures using TypeScript syntax instead
+              of cumbersome and error-prone service-specific domain specific
+              languages.
+            </p>
+          </div>
+          <div className="mt-10">
+            <h5 className="m-0">Automated IAM policies</h5>
+            <p className="body1 text-functionless-medium dark:text-functionless-dark-medium mt-2">
+              The best security auditor is no auditor - our compiler derives
+              minimal IAM Policies from your code.
+            </p>
+          </div>
```

Comment:
Should we create a component for feature

Reply:
definitely